libvast/src/command.cpp

```diff
@@ -353,10 +353,11 @@ parse(const command& root, command::argument_iterator first,
 
 bool init_config(caf::actor_system_config& cfg, const command::invocation& from,
                  std::ostream& error_output) {
-  // Adjust default file verbosity for commands starting a node,
+  // Disable file logging for all commands that don't start a node,
   // i.e., `vast start` and `vast -N ...`.
-  if (from.name() == "start" || caf::get_or(from.options, "system.node", false))
-    cfg.set("logger.file-verbosity", defaults::logger::server_file_verbosity);
+  if (!(from.name() == "start"
+        || caf::get_or(from.options, "system.node", false)))
+    cfg.set("logger.file-verbosity", caf::atom("quiet"));
```

Comment:
Did you verify that this doesn't create log files for non-server commands? I faintly remember that the log level should be 'quiet' *before* caf is initialized, otherwise I did end up with a log file containing a single line printed during vast startup. But maybe this has already changed in caf in the meantime.

Reply:
This code runs before CAF is initialized. I just checked to make sure it behaves at expected.